lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -635,23 +710,27 @@ defmodule ExUnitTest do
 
         @tag :skip
         test "skipped", do: assert(true)
+
         test "pass #{__ENV__.line}", do: assert(true)
         test "pass #{__ENV__.line}", do: assert(true)
         test "fail #{__ENV__.line}", do: assert(false)
         test "fail #{__ENV__.line}", do: assert(false)
+
+        @tag :exclude
+        test "exclude me please", do: assert(true)
```

Comment:
Nice! :+1: We should probably add `:exclude` to all other `:max_failures` test cases here, like we do with `@tag :skip`.

Reply:
I will leave that for another PR once this is merged

Reply:
Your call. But since we are fixing tests related to the exclusion of exclude, adding those now may help us find more failures.

Reply:
I will add it then

Reply:
done